backend/ibutsu_server/db/upgrades.py

```diff
@@ -50,3 +51,24 @@ def upgrade_1(session):
             "owner_id",
             schema=Column(PortableUUID(), ForeignKey("users.id"), index=True),
         )
+
+
+def upgrade_2(session):
+    """Version 2 upgrade
+
+    This upgrade adds indices for the metadata.tags fields in the results/runs tables.
+
+    It is equivalent to running the following SQL statements:
+        CREATE INDEX IF NOT EXISTS ix_runs_tags ON runs USING gin ((data->'tags'));
+        CREATE INDEX IF NOT EXISTS ix_results_tags ON results USING gin ((data->'tags'));
+    """
+    TABLES = ["runs", "results"]
+
+    op = get_upgrade_op(session)
+    for table in TABLES:
```

Comment:
I think we probably want to just check that the engine is postgres before running this. Just in case someone decides to run this locally using SQLite or something.

Reply:
Ah, good idea, updated to check the database type.